### PR TITLE
Y24-265 - RNA FFPE submission

### PR DESCRIPTION
**Pull request by BenTopping**
#### Changes proposed in this pull request

- Adds RNAF Submissions (Library Prep, Library Prep + pool, NovaSeqX, NovaSeq 6000, MiSeq)
- Adds RNA FFPE library type
- Adds library_rnaf request type
- Adds RNAF product catalogue

#### Notes

There is an associated PR in the integration suite for the RNA FFPE spec.

Limber PR: https://github.com/sanger/limber/pull/2004

### CODE REVIEW

File: config/default_records/submission_templates/015_rnaf_submission_templates.wip.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    product_line_name: Illumina-HTP
+    product_catalogue_name: RNAF
+    request_type_keys: ["limber_rnaf", "limber_multiplexing", "illumina_htp_novaseq_6000_paired_end_sequencing"]
+Limber-Htp - RNAF - MiSeq sequencing:
+  name: "Limber-Htp - RNAF - MiSeq sequencing"
+  submission_class_name: "LinearSubmission"
+  related_records:
+    product_line_name: Illumina-HTP
+    product_catalogue_name: RNAF
+    request_type_keys: ["limber_rnaf", "limber_multiplexing", "illumina_b_miseq_sequencing"]
```
Comment (KatyTaylor) [inline config/default_records/submission_templates/015_rnaf_submission_templates.wip.yml:33]: "will be a Single Library only submission for reISC"

Does this mean we only need the "Limber-Htp - RNAF" template, or do we need all these other ones too?

File: config/default_records/request_types/004_limber_high_throughput.yml [hunk 2]
```diff
@@ -199,3 +199,10 @@ limber_targeted_nanoseq:
     - LTN Cherrypick
   library_types:
     - Targeted Nanoseq
+limber_rnaf:
+  <<: *limber_htp_library
+  name: Limber RNAF
+  acceptable_purposes:
+    - LBR Cherrypick
+  library_types:
+    - RNA FFPE
```
Comment (KatyTaylor) [inline config/default_records/request_types/004_limber_high_throughput.yml:208]: "May need to inform NPG about this" - reminder - has this been done?

**Review by KatyTaylor — COMMENTED** [review]
Looks good. I've just compared against the requirements on the story and left a couple of questions (more on the Limber side in a sec). I suspect you may have had some conversations with Liz already to clarify some of these things.

**Comment by KatyTaylor** [comment]
Oh and test failing - haven't checked it out.

File: config/default_records/request_types/004_limber_high_throughput.yml (hunk 2 above)
Comment (BenTopping) [reply config/default_records/request_types/004_limber_high_throughput.yml:208]: Nope, good reminder thanks.

File: config/default_records/submission_templates/015_rnaf_submission_templates.wip.yml (hunk 1 above)
Comment (BenTopping) [reply config/default_records/submission_templates/015_rnaf_submission_templates.wip.yml:33]: Liz has request the same templates available for the other rna pipelines as well, so multiplexing and sequencing requests are required.

**Review by KatyTaylor — APPROVED** [review]

**Pull request merged**